addon/inspect.js

```diff
@@ -1451,7 +1459,7 @@ class FieldValueCell extends React.Component {
     let {row} = this.props;
     if (row.tryEdit()) {
       let td = e.currentTarget;
-      row.rowList.model.didUpdate(() => td.querySelector("textarea").focus());
+      row.rowList.model.didUpdate(() => td.querySelector("#input-field").focus());
```

Comment:
Instead of using querySelector, could you use refs ? 
```suggestion
      row.rowList.model.didUpdate(() => this.refs.inputField.focus());
```
You can find other examples in the code